Makefile

```diff
@@ -89,24 +72,17 @@ kind: images
 	kind load docker-image onosproject/model-compiler:${MODEL_COMPILER_VERSION}
 
 publish: # @HELP publish version on github (called by release-merge-commit)
-	./../build-tools/publish-version ${VERSION} onosproject/model-compiler
+	./build/build-tools/publish-version ${VERSION} onosproject/model-compiler
 
-jenkins-publish: build-tools jenkins-tools docker-login # @HELP Jenkins calls this to publish artifacts
+jenkins-publish: docker-login # @HELP Jenkins calls this to publish artifacts
 	make -C models/ric-1.x publish
 	make -C models/e2node-1.x publish
 	make -C models/devicesim-1.0.x publish
 	make -C models/testdevice-1.0.x publish
 	make -C models/testdevice-2.0.x publish
-	../build-tools/release-merge-commit
+	./build/build-tools/release-merge-commit
 
-clean: # @HELP remove all the build artifacts
+clean:: # @HELP remove all the build artifacts
```

Comment:
extra `:`?

Reply:
The base common.mk defines a clean rule already. This is the make syntax to add to it.